modules/collect/src/main/java/com/opengamma/strata/collect/result/FailureItem.java

```diff
@@ -129,11 +131,11 @@ static FailureItem of(FailureReason reason, String message, int skipFrames) {
    * 
    * @param reason  the reason
    * @param message  the failure message, not empty
-   * @param attributes the attributes associated to this failure
+   * @param attributes the attributes associated with this failure
    * @param skipFrames  the number of caller frames to skip, not including this one
    * @return the failure
    */
-  static FailureItem of(FailureReason reason, String message, Map<String, String> attributes, int skipFrames) {
+  public static FailureItem of(FailureReason reason, String message, Map<String, String> attributes, int skipFrames) {
```

Comment:
Get rid of `skipFrames`, it's in internal implementation detail. Have a look at the other public `of` methods to see what they do.